packages/auth/amplify_auth_cognito_ios/ios/Classes/FlutterSignInRequest.swift

```diff
@@ -36,16 +36,4 @@ struct FlutterSignInRequest {
      )
      return AuthSignInOperation.Request.Options(pluginOptions: pluginOptions)
   }
-
```

Comment:
If we're gonna remove this then please remove the force casts above since they will crash the app if ever triggered

Reply:
Done